src/components/store/types/ActionType.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Available action types
+ */
+export enum ActionType {
```

Comment:
It looks similar with the [BlockMutationType](https://github.com/codex-team/editor.js/blob/next/types/events/block/mutation-type.ts)

Could we use the same type in both cases?